scripts/generate_subsections_ids.py

```diff
@@ -0,0 +1,86 @@
+######
```

Comment:
I don't think we need to upload this script as we won't need to run it on a regular basis. @yuleib - Do we have a repo of scripts we want to save? If so let's add it there (and if not now then before merging to dev)